tests/order_modifiers_integration/_bootstrap.php

```diff
@@ -17,6 +18,8 @@
 $ticketable[] = 'post';
 tribe_update_option( 'ticket-enabled-post-types', array_values( array_unique( $ticketable ) ) );
 
+define( 'JSON_SNAPSHOT_OPTIONS', JSON_PRETTY_PRINT | JSON_UNESCAPED_UNICODE | JSON_UNESCAPED_SLASHES );
```

Comment:
What does this do?

Reply:
Controls the default output of JSON snapshot. It makes the beautiful and easy to read pretty much.

Whenever you use snapshot assertion you can use the method `assertMatchesJsonSnapshot`